cmd/pb/main.go

```diff
@@ -182,4 +182,20 @@ func addServiceGenerateCommands(rootCmdGenerate *cobra.Command) {
 	cmdErrorDefinitions.Flags().StringVar(&errorsDefinitionsPath, "path", "", "path for generated file")
 	cmdErrorDefinitions.Flags().StringVar(&errorsDefinitionsSource, "source", "", "JSONAPI conform error definitions source to use for generation")
 	rootCmdGenerate.AddCommand(cmdErrorDefinitions)
+	addErrorDefinitionsCommands(cmdErrorDefinitions)
```

Comment:
There is already a command for creating errors-definitions. I would prefer to re-use this with an output switch e.g. -o go or -o md.

Reply:
refactored